packages/account/src/Components/trading-assessment/trading-assessment-radio-buttons.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Field } from 'formik';
+import { Text, RadioGroup } from '@deriv/components';
+
+const TradingAssessmentRadioButton = ({ text, list, onChange, values, form_control, setEnableNextSection }) => {
+    React.useEffect(() => {
+        setEnableNextSection(!!values[form_control]);
+    }, [form_control]);
+
+    return (
+        <div className='trading-assessment__wrapper__question'>
+            <Text as='h1' color='prominent' weight='bold' size='xs'>
+                {text}
+            </Text>
+            <Field name={form_control}>
+                {() => (
+                    <RadioGroup
+                        className='trading-assessment__wrapper__question--radio-group'
+                        is_left
+                        should_wrap_items
+                        required
+                        selected={values[form_control]}
+                        onToggle={e => {
+                            onChange(e);
+                            setEnableNextSection(true);
+                        }}
+                    >
+                        {list.map(answer => (
+                            <RadioGroup.Item
+                                className='trading-assessment__wrapper__question--radio-group--item'
+                                key={answer.value}
```

Comment:
should we add change this to   key={answer?.value}

Reply:
The list is static and embedded in the code. So its not necessary